ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx

```diff
@@ -171,6 +177,38 @@ export class OntotextYasguiWebComponent {
     this.showSaveQueryDialog = false;
   }
 
+  /**
+   * Flag controlling the visibility of the saved queries list popup.
+   */
+  @State() showSavedQueriesPopup = false;
+
+  /**
+   * Handler for the event fired when the show saved queries button in the yasqe toolbar is triggered.
+   */
+  @Listen('internalShowSavedQueriesEvent')
+  showSavedQueriesHandler() {
+    this.loadSavedQueries.emit(true);
+  }
+
+  /**
+   * Handler for the event fired when the saved query is selected from the saved queries list.
+   */
+  @Listen('internalSaveQuerySelectedEvent')
+  savedQuerySelectedHandler(event: CustomEvent<SaveQueryData>) {
+    const queryData: SaveQueryData = event.detail;
+    this.config.savedQueries.data = [];
```

Comment:
IMO. Would be better if we not change configuration. The configuration is responsibility of the client. If we start to touch it will be more complex to understand who change the config we or client. We just have to fire event when button is clicked and if client want will change configuration. If some clients do not have dynamic creation of saved query, they will setup queries on initialization and will not need to pass same data again.